build.gradle

```diff
@@ -181,6 +181,8 @@ dependencies {
 
     // ZipArchive dependencies used for integration tests
     zipArchive group: 'org.opensearch.plugin', name:'opensearch-ml-plugin', version: "${opensearch_build}"
+    zipArchive group: 'org.opensearch.plugin', name:'opensearch-knn', version: "${opensearch_build}"
+    zipArchive group: 'org.opensearch.plugin', name:'neural-search', version: "${opensearch_build}"
```

Comment:
Do we want to commit this?

Reply:
Yes it was added for IT testing, if I can't get BWC to work though I will remove this and the IT I added and add that in a different PR.